solr/core/src/java/org/apache/solr/handler/export/ExportWriter.java

```diff
@@ -147,10 +143,20 @@ public String getContentType() {
 
   @Override
   public void close() throws IOException {
-    if (writer != null) writer.close();
+    if (writer != null) {
+      try {
+        writer.close();
+      } catch (Throwable t) {
+        //We're going to sit on this.
+      }
+    }
     if (respWriter != null) {
-      respWriter.flush();
-      respWriter.close();
+      try {
+        respWriter.flush();
+        respWriter.close();
+      } catch (Throwable t) {
+
```

Comment:
Can we really suppress any exceptions that occur here without even logging them out?

If this catch block is triggered frequently during normal execution and we don't want to clutter the logs, I appreciate that.  But would there be any value in logging the exception at TRACE or some other obscure level, so that someone can get it printed out if they really need it?